syntaxes/yaml.tmLanguage.json

```diff
@@ -248,7 +248,7 @@
 					"name": "invalid.illegal.character.anchor.yaml"
 				}
 			},
-			"match": "((\\*))([^\\s\\[\\]\/{\/},]+)([^\\s\\]},]\\S*)?"
+			"match": "((\\*))([^\\s\\[\\]/{/},]+)([^\\s\\]},]\\S*)?"
```

Comment:
Can you confirm that this still works as expected? This looks like an autofix. 
It might work fine but it is better to be intentional.

Reply:
Yes that changed by prettier, i'm going to add this file to ignored for prettier